From: Adam Conrad <adconrad@0c3.net>
Date: Thu, 4 Jun 2026 13:32:13 +0100
Subject: Fix up FHS file locations for apache2 droppings.

Forwarded: not-needed
Reviewed-By: Yadd <yadd@debian.org>
Last-Update: 2023-10-19
---
 configure.in    | 6 +++---
 include/httpd.h | 4 ++--
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/configure.in b/configure.in
index 3f51b79..1e10e34 100644
--- a/configure.in
+++ b/configure.in
@@ -1058,11 +1058,11 @@ AC_MSG_NOTICE([])
 APACHE_GEN_CONFIG_VARS
 
 APR_EXPAND_VAR(ap_prefix, $prefix)
-AC_DEFINE_UNQUOTED(HTTPD_ROOT, "${ap_prefix}",
+AC_DEFINE_UNQUOTED(HTTPD_ROOT, "/etc/apache2",
 	[Root directory of the Apache install area])
-AC_DEFINE_UNQUOTED(SERVER_CONFIG_FILE, "${rel_sysconfdir}/${progname}.conf",
+AC_DEFINE_UNQUOTED(SERVER_CONFIG_FILE, "${progname}.conf",
 	[Location of the config file, relative to the Apache root directory])
-AC_DEFINE_UNQUOTED(AP_TYPES_CONFIG_FILE, "${rel_sysconfdir}/mime.types",
+AC_DEFINE_UNQUOTED(AP_TYPES_CONFIG_FILE, "mime.types",
 	[Location of the MIME types config file, relative to the Apache root directory])
 
 perlbin=`$ac_aux_dir/PrintPath perl`
diff --git a/include/httpd.h b/include/httpd.h
index 5db96fb..67f5f3d 100644
--- a/include/httpd.h
+++ b/include/httpd.h
@@ -108,10 +108,10 @@ extern "C" {
 #ifndef DOCUMENT_LOCATION
 #ifdef OS2
 /* Set default for OS/2 file system */
-#define DOCUMENT_LOCATION  HTTPD_ROOT "/docs"
+#define DOCUMENT_LOCATION "/var/www/html"
 #else
 /* Set default for non OS/2 file system */
-#define DOCUMENT_LOCATION  HTTPD_ROOT "/htdocs"
+#define DOCUMENT_LOCATION "/var/www/html"
 #endif
 #endif /* DOCUMENT_LOCATION */
 
