From: Adam Conrad <adconrad@0c3.net>
Date: Thu, 4 Jun 2026 13:32:13 +0100
Subject: Remove LD_LIBRARY_PATH from envvars-std

Forwarded: no
Last-Update: 2012-04-15
---
 support/envvars-std.in | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/support/envvars-std.in b/support/envvars-std.in
index 9493bc7..2bb3412 100644
--- a/support/envvars-std.in
+++ b/support/envvars-std.in
@@ -18,11 +18,4 @@
 #
 # This file is generated from envvars-std.in
 #
-if test "x$@SHLIBPATH_VAR@" != "x" ; then
-  @SHLIBPATH_VAR@="@exp_libdir@:$@SHLIBPATH_VAR@"
-else
-  @SHLIBPATH_VAR@="@exp_libdir@"
-fi
-export @SHLIBPATH_VAR@
-#
 @OS_SPECIFIC_VARS@
